### PR TITLE
Add HTTP Status Codes guide

### PR DESCRIPTION
**Pull request by rb455**
Adds a guide inspired by [Centrapay's guide](https://docs.centrapay.com/api/http-status-codes/) but with the use of different wording.

Ticket Link: this is an [action item](https://www.notion.so/immersve/Get-HTTP-status-codes-documented-similar-to-https-docs-centrapay-com-api-http-status-codes-d408f9f47b3e408f8aa9a6e31cbbb795?pvs=4) from tech catch up 
doc design entry: https://www.notion.so/immersve/Docs-Content-Design-858f8fda3f1442ea82f8ddc8f007d792?p=28634570a3ec42e09d4a282541a24801&pm=s

### CODE REVIEW

File: imsv-docs-astro/src/content/docs/guides/http-status-codes.mdoc [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- Check the Retry-After HTTP response header for the number of seconds before
+the next request will be accepted.
+- Contact [support@immersve.com](mailto:support@immersve.com) to increase your
+limits.
+
+## 5xx Server Error
+Errors in the 5xx range suggest a problem on our side. Usually, retrying your
+request might resolve the issue. Our engineering team monitors these errors
+closely but bug reports are also welcome at
+[support@immersve.com](mailto:support@immersve.com)
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/http-status-codes.mdoc:78]:
```suggestion
[support@immersve.com](mailto:support@immersve.com).
```

File: imsv-docs-astro/src/content/docs/guides/http-status-codes.mdoc [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+## 401 Unauthorized
+API key or JWT is missing, expired or invalid. Refer to our [Authentication
+guide](/guides/authentication) for more information.
+
+## 403 Forbidden
+Resource is missing, permission is denied or there is a business rule
+violation.
+
+### Resource missing or permission denied
+- Check the resource id is correct
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/http-status-codes.mdoc:38]:
```suggestion
- Check the resource id is correct.
```

File: imsv-docs-astro/src/content/docs/guides/http-status-codes.mdoc [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+resource you are accessing.
+- Check your user or API key's roles for the account allow the required action.
+
+### Business rule violated
+In cases where access to a resource is granted but a request is still forbidden
+due to a business rule violation, a 403 status code is returned with additional
+details in the `errorCode` field of the response body. Specific `errorCode`
+values and their meanings will be documented on each endpoint.
+
+{% code title="Example response" %}
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/http-status-codes.mdoc:49]: The response example is wrong. 

```yaml

  statusCode: 403
  statusName: 'Forbidden'
  errorCode: ...
  reason: ...

```

**Review by ncjones — DISMISSED** [review]

File: imsv-docs-astro/src/content/docs/guides/http-status-codes.mdoc (hunk 3 above)
Comment (rb455) [reply imsv-docs-astro/src/content/docs/guides/http-status-codes.mdoc:49]: good catch, thanks! updated

**Review by ncjones — APPROVED** [review]

**Pull request merged**